plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcWriteSessionProperties.java

```diff
@@ -49,9 +50,14 @@ public JdbcWriteSessionProperties(JdbcWriteConfig writeConfig)
                         false))
                 .add(booleanProperty(
                         NON_TRANSACTIONAL_INSERT,
-                        "Do not use temporary table on insert to table",
```

Comment:
It was modified by mistake, actually should update the description for `NON_TRANSACTIONAL_MERGE`

Reply:
sent https://github.com/trinodb/trino/pull/24431